.github/workflows/test-action.yml

```diff
@@ -35,6 +35,16 @@ jobs:
         run: |
           pip3 install nox
 
+      - name: Install Linux dependencies
```

Comment:
https://github.com/saltstack/salt/blob/master/.github/actions/setup-shellcheck/action.yml ?

Reply:
That'd be great to have someplace where we can re-use it, for now I'll proceed.